pkg/awsutils/awsutils.go

```diff
@@ -1039,14 +1048,22 @@ func (cache *EC2InstanceMetadataCache) DescribeAllENIs() ([]ENIMetadata, map[str
 		if interfaceType == "trunk" {
 			trunkENI = eniID
 		}
+		if interfaceType == "efa" {
```

Comment:
Can we consider to use switch/case to evaluate interfaceTypes?

Reply:
If we get a third custom interface type, I think it's about time to switch. 😀